gateway/testutil.go

```diff
@@ -233,6 +233,9 @@ func (r *ReloadMachinery) TickOk(t *testing.T) {
 func InitTestMain(ctx context.Context, m *testing.M) int {
 	test.InitTestMain(ctx, m)
 
+	BackoffMultiplier = 0
```

Comment:
we could add these to `config.Private` and control it while during gateway startup

Reply:
as discussed, no change for now; agreement that this may be configurable, possibly even from gateway config (not just private), or future refactor. Replacing a coupling with another coupling wouldn't make much difference right now, but we definitely don't want globals polluting gateway long term.